### PR TITLE
fix($browser): Added e.preventDefault() to AutoCompleteItem touch event

### PR DESCRIPTION
**Pull request by spykr**
Fixed a bug where touching a suggested address would trigger any clickable elements beneath it

Closed issue #123

### CODE REVIEW

**Comment by cscottsa** [comment]
@kenny-hibino can we please look into merging this please?

File: src/PlacesAutocomplete.js [hunk 1]
```diff
@@ -74,7 +74,10 @@ class PlacesAutocomplete extends Component {
     this.setState({ autocompleteItems: [] })
   }
 
-  selectAddress(address, placeId) {
+  selectAddress(address, placeId, e) {
+    if (e !== undefined) {
```
Comment (kenny-hibino) [inline src/PlacesAutocomplete.js]: @spykr When will this event `e` be `undefined`?

**Comment by BenBish** [comment]
@kenny-hibino please can you consider merging this?

**Comment by joenot443** [comment]
I would also appreciate if @kenny-hibino merged this, I'm still getting the bug.

**Comment by DevJS** [comment]
👍 please merge :)

**Comment by conorpirie** [comment]
Please merge, I really need this fix! 👍

**Comment by brendonlamb** [comment]
@conorpirie @DevJS 

Temporary solution if you need: 
https://www.npmjs.com/package/fastclick

Literally just install that and everything gets better :) I was having similar (click through) problems with another package... this fixed the my whole app. Might consider it for my bootstrap :)

**Comment by kenny-hibino** [comment]
Merged with ef4dc4629bbc9fa3245bc30aadcd1c05d29a1baa

Thank you @spykr 🎉

**Comment by DevJS** [comment]
Thanks a lot! Don't forget to build the package by running `npm run build` so that everyone can `npm install` :)

**Comment by kenny-hibino** [comment]
@DevJS Are you having trouble installing the package?

**Comment by DevJS** [comment]
The package we are installing from npm is the built version of the code apparently, so you might have to run `npm run build` and publish the update then!

**Pull request closed without merging**